src/Dot/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { stringToColor } from '../util/colorMapping';
+import { Props } from './props';
+import { styles } from './styles';
+
+export const Dot = ({
+  avatarUrl = undefined,
+  username = '',
+  style = {},
+  visible = false
```

Comment:
Maybe we should default this to true?

Would be weird if somebody implements it and nothing shows up haha